src/Features/Core/Portable/Diagnostics/Log/DiagnosticAnalyzerLogger.cs

```diff
@@ -179,18 +164,5 @@ private static bool CheckTelemetry(DiagnosticAnalyzer analyzer, IDiagnosticAnaly
             var diagnostic = diagDescriptors.Length > 0 ? diagDescriptors[0] : null;
             return diagnostic == null ? false : diagnostic.CustomTags.Any(t => t == WellKnownDiagnosticTags.Telemetry);
         }
-
-        private static SHA256 GetSha256()
-        {
-            try
-            {
-                // not all environment allows SHA256 encryption
```

Comment:
The case can't happen since SHA256 is supported everywhere. We previously used a different class that didn't. Instead of replacing it with SHA256 we added try-catch :(.